.github/workflows/scheduled.yaml

```diff
@@ -96,3 +96,24 @@ jobs:
                   
           # Running juju status will show us the current revisions of the charms.
           juju status --relations
+
+      - name: Check Legend Public Instance status
+        run: |
+          wait_for_curl() {
+            url="$1"
+            for i in {1..30}; do
+              # Any request to any Legend Application should be redirected to gitlab.com
+              curl -i "${url}" | grep "302 Found" && break
+              if [ "$i" -eq 30 ]; then
+                echo "Failed to get '302 Found' response status from ${url}"
+                exit 1
+              fi
+              sleep 10
+            done
+          }
+
+          wait_for_curl "https://juju-acct.legend.finos.org/"
+          wait_for_curl "https://juju-acct.legend.finos.org/api"
+          wait_for_curl "https://juju-acct.legend.finos.org/legend"
```

Comment:
We should check for ``/``, ``/api``, and ``/engine``.

Reply:
'/' and '/api' are checked. I will add '/engine'